.ruff.toml

```diff
@@ -37,15 +40,19 @@ fixable = [
     "D", # pydocstyle
 ]
 
+src = ["src", "tests", "examples"]
```

Comment:
"scripts" could also be included here

Reply:
I agree and will address these items in the upcoming ticket for the template, but these changes were more pressing.